scripts/Tools/code_checker

```diff
@@ -107,6 +107,8 @@ def check_code(dir_to_check, num_procs, files):
         files_to_check = [item for item in files_to_check if matches(item, files)]
     else:
         files_to_check = [item for item in files_to_check if item.endswith(".py")]
+    if len(files_to_check) == 0:
```

Comment:
Pattern: foo.py -> Goes through the matches check
Path: ./foo.py, /home/jgfouca/foo.py -> Goes directly into files_to_check